packages/paste-style-props/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@twilio-paste/style-props",
```

Comment:
I know we discussed shoving these into types, but honestly it just didn't feel right, and this feels more appropriate. Even pulling in the types for style-props into here felt better.